gistUpload.js

```diff
@@ -14,15 +14,14 @@ const HANG_PERCENT = 0.25;
  * @param {string} gist - Name of gist supplied by user
  */
 function uploadFile(contents, gist) {
-  const post = { [gist]: { content: contents } };
   return axios({
     method: 'post',
     url: API_URL,
     auth: {
       user: process.env.USER,
       password: process.env.SECRET_TOKEN,
     },
-    data: { files: post },
+    data: { files: { [gist]: { content: contents } } },
```

Comment:
rename `contents` argument to `content` so that you can say `data: { files: { [gist]: { content } } }`